### PR TITLE
Fix crash when actors with multiple RevealsShroud target a dead frozen actor.

### PR DESCRIPTION
**Pull request by pchote**
This should fix the crash that @SoScared and co encountered in their gapgen playtest stream.

### CODE REVIEW

File: OpenRA.Mods.Common/ActorExts.cs
Comment (Phrohdoh) [inline OpenRA.Mods.Common/ActorExts.cs]: Does it make sense to hardcode this value?

**Review by GraionDilach — APPROVED** [review]

**Comment by pchote** [comment]
This is blocking @SoScared being able to make balance tests for the active gap generators, so can we please get a second review and merge here?

File: OpenRA.Mods.Common/ActorExts.cs
Comment (obrakmann) [inline OpenRA.Mods.Common/ActorExts.cs]: The only problem I can see with this is that civilian would be stopping short when ordered to attack a frozen actor, since they only have 1 sight range. I doubt that that would ever be an issue, though.

**Review by obrakmann — APPROVED** [review]
👍

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0569f9a9cdfc3175d10183414d2171a1efe0887f)

**Pull request merged**
